plugins/Dashboard/javascripts/widgetMenu.js

```diff
@@ -35,10 +60,33 @@ widgetsHelper.getAvailableWidgets = function () {
  * Returns the complete widget object by its unique id
  *
  * @param {string} uniqueId
+ * @param callback
  * @return {object} widget object
  */
-widgetsHelper.getWidgetObjectFromUniqueId = function (uniqueId) {
-    var widgets = widgetsHelper.getAvailableWidgets();
+widgetsHelper.getWidgetObjectFromUniqueId = function (uniqueId, callback) {
+    if(!callback) { // @deprecated
+        return this.getWidgetObjectFromUniqueIdSync(uniqueId);
```

Comment:
Same here and other methods. We can break previous behaviour unless we still use it somewhere ourselves